aws/fake/cf.go

```diff
@@ -80,9 +86,8 @@ func MockCSOutput(stackId string) *cloudformation.CreateStackOutput {
 }
 
 func (m *CFClient) UpdateStack(params *cloudformation.UpdateStackInput) (*cloudformation.UpdateStackOutput, error) {
-	m.lastStackTags = params.Tags
-	m.lastStackParams = params.Parameters
-	m.lastStackTemplate = *params.TemplateBody
+	// TODO: Update stack needs another logic to register state update, so createStack and updateStack don't
+	// mess with each other states.
```

Comment:
is this a TODO for the next PR or should be cleaned up?

Reply:
I don't have a specific next PR planned but I believe this is a useful reminder for future test implementations.

This is an area that is not tested yet. We only test creation we still don't test updates. The reason I left the TODO there is to remind in next iterations that update should not use the same variables create uses to store state to avoid bugs.

Reply:
Ok, thanks!
Maybe create an issue and link from the TODO to the issue?

Reply:
Done: https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/653